src/main/steps/tasklistresponse/support-you-need-during-case/court-hearing-support/content.ts

```diff
@@ -140,6 +149,9 @@ export const form: FormContent = {
               type: 'textarea',
               label: l => l.otherDetails,
               labelSize: null,
+              attributes: {
+                rows: 2,
```

Comment:
we dont have a character limitation for text area, defining row numbers is not required. Please confirm with BA's about the character limitation.

Reply:
text area - display should be same as shown in the prototype
text area- number of characters we can enter is unlimited…
as per BA
will update the code as required